app/Http/Controllers/CompanyMaster/PersonnelController.php

```diff
@@ -8,12 +8,28 @@
 
 class PersonnelController extends Controller
 {
+
+    /**
+     * 要員のサジェスト機能 viewの表示
+     * @return view
+     */
+    public function index()
+    {
+      return view('agreement.contractList');
+    }
+
      /**
      * 要員のサジェスト機能
      * @return view
      */
-    public function showIndex()
+    public function autocomplete(Request $request)
     {
-
+      $term = $request->get('term');
+      $personnels = Personnel::where('personnel_name', 'like', '%' .$term. '%')->get();
```

Comment:
三原さんの時にちょっと話題に出しましたが
like検索するなら'personnel_nameにインデックス張っちゃいます…？
`張ったら張ったで`前方一致じゃないと恩恵はあまり受けられませんが…

Reply:
```suggestion
      $personnels = Personnel::where('personnel_name', 'like', $term. '%')->get();
```
インデックス張ったなら前方一致検索にした方が良いです。